### PR TITLE
Single page styling

### PR DESCRIPTION
**Pull request by kfalkowski11**
#### Overview
Single page styling updates
* All inputs and selects have rounded corners and floating labels
* All buttons fixed to match figma mocks
* All links updated to same style
* All dividers changed to lighter grey
* Error messages have rounded borders
* Product images have rounded border with lighter grey
* Summary added to Inventory and Confirmation pages
* Summary styling changed on mobile to inline instead of slide up from bottom
* Add header with logo
* Add ability to switch languages
* Minor spacing fixes

Inventory page was also outdated since the Inventory hook updates, updated code to match one click.

Videos of all checkout flows can be found here: https://drive.google.com/drive/folders/1cKo2itbd8QTX1v15WJt0CDig7UrMsOcm?usp=sharing

### CODE REVIEW

File: single_page/src/pages/ConfirmationPage/ConfirmationPage.jsx [hunk 1]
```diff
@@ -15,6 +16,10 @@ const ConfirmationPage = () => {
   const payments = state.applicationState.payments;
   const { t } = useTranslation();
 
+  const continueShopping = () => {
+
+  };
```
Comment (TimeBones) [inline single_page/src/pages/ConfirmationPage/ConfirmationPage.jsx:21]: Should return to shop url?

Comment (kfalkowski11) [reply single_page/src/pages/ConfirmationPage/ConfirmationPage.jsx:21]: Good catch!

**Comment by TimeBones** [comment]
Adding an invalid discount causes the apply button to be taller than the discount input field.
image

File: single_page/src/components/Customer/Customer.jsx [hunk 2]
```diff
@@ -17,41 +18,55 @@ const MemoizedCustomer = memo(({ customer, submitCustomer }) => {
   const logError = useErrorLogging();
   const [email, setEmail] = useState(customer?.email_address);
   const [errors, setErrors] = useState(null);
-  const [loading, setLoading] = useState(false);
+  const [ acceptsMarketing, setAcceptsMarketing ] = useState(false)
   const { t } = useTranslation();
+  console.log("customer", customer);
```
Comment (TimeBones) [inline single_page/src/components/Customer/Customer.jsx:23]: Left in log

**Pull request merged**